WordPress/src/main/java/org/wordpress/android/ui/mysite/QuickStartCardViewHolder.kt

```diff
@@ -0,0 +1,114 @@
+            quick_start_card_recycler_view.apply {
+                adapter = DummyTaskAdapter()
+                layoutManager = LinearLayoutManager(context, HORIZONTAL, false)
+                setRecycledViewPool(viewPool)
+                addItemDecoration(RecyclerItemDecoration(DisplayUtils.dpToPx(context, 8), 0))
+            }
+        }
+    }
+
+    fun bind(item: QuickStartCard) = itemView.apply {
+        ObjectAnimator.ofInt(quick_start_card_progress, "progress", item.progress).setDuration(600).start()
+
+        val progressTrackColor = ContextCompat.getColor(context, item.progressColor.trackColor)
+        val progressIndicatorColor = ContextCompat.getColor(context, item.progressColor.indicatorColor)
+        if (Build.VERSION.SDK_INT > Build.VERSION_CODES.LOLLIPOP) {
+            quick_start_card_progress.progressBackgroundTintList = ColorStateList.valueOf(progressTrackColor)
+            quick_start_card_progress.progressTintList = ColorStateList.valueOf(progressIndicatorColor)
+        } else {
+            // Workaround for Lollipop
+            val progressDrawable = quick_start_card_progress.progressDrawable.mutate() as LayerDrawable
+            val backgroundLayer = progressDrawable.findDrawableByLayerId(android.R.id.background)
+            val progressLayer = progressDrawable.findDrawableByLayerId(android.R.id.progress)
+            backgroundLayer.colorFilter = createBlendModeColorFilterCompat(progressTrackColor, SRC_IN)
+            progressLayer.colorFilter = createBlendModeColorFilterCompat(progressIndicatorColor, SRC_IN)
+            quick_start_card_progress.progressDrawable = progressDrawable
+        }
+
+        quick_start_card_title.text = item.title
+        (quick_start_card_recycler_view.adapter as? DummyTaskAdapter)?.loadData(item.tasks)
+        quick_start_card_more_button.setOnClickListener { item.onMoreClick?.click() }
+    }
+}
+
+class DummyTaskAdapter : Adapter<DummyTaskViewHolder>() {
+    private var items = listOf<DummyTask>()
+
+    fun loadData(newItems: List<DummyTask>) {
+        val diffResult = DiffUtil.calculateDiff(DummyTaskAdapterDiffCallback(items, newItems))
+        items = newItems
+        diffResult.dispatchUpdatesTo(this)
+    }
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int) = DummyTaskViewHolder(
+            LayoutInflater.from(parent.context).inflate(R.layout.quick_start_task_card, parent, false)
+    )
+
+    override fun onBindViewHolder(holder: DummyTaskViewHolder, position: Int) {
+        holder.bind(items[position])
+    }
+
+    override fun getItemCount() = items.size
+
+    inner class DummyTaskViewHolder(itemView: View) : ViewHolder(itemView) {
+        fun bind(task: DummyTask) = itemView.apply {
+            dummy_task_title.text = task.title
+            dummy_task_description.text = task.description
+
+            val alpha = if (task.done) 0.2f else 1.0f
```

Comment:
Are these temporary or should we move them to constants now?

Reply:
We should probably move these to constants, well noted!